<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QMainWindow" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget2">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>-10</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background color: black;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="menupage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>38</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>MAIN MENU </string>
      </property>
     </widget>
     <widget class="QPushButton" name="addbutton1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>ADD PRODUCT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updatedeletebutton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>UPDATE/DELETE PRODUCT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewbutton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>230</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>VIEW PRODUCT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loggoutbutton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>390</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>LOGOUT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createbutton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>310</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CREATE BILL</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="addpage">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ADD PRODUCT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameinput2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="priceinput2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>300</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product Price:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idinput2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product Id:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>360</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product Quantity:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="qntyinput2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Normal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="addbutton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>470</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>520</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back To Main Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="updatepage">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>401</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>UPDATE/DELETE PRODUCT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>300</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product Price:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameinput">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>240</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="priceinput">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>340</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="updatebutton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>470</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletebutton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>480</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idinput">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>390</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product quantity</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="qntyinput">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>420</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="back1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>510</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back To Main Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="view1">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>VIEW PRODUCT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameinput3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="viewbutton_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>410</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idinput3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product Id:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>480</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back To Main Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="view2">
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>VIEW PRODUCT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameoutput">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>200</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product price:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>330</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product in stock:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="priceoutput">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>280</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="stockoutput">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>360</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>440</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idoutput">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>130</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product ID:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back3">
      <property name="geometry">
       <rect>
        <x>379</x>
        <y>540</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back To Main Menu</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
